docs/csi-controller-attach-detach.md

```diff
@@ -0,0 +1,123 @@
+# Restricting REST access with Pure Service Orchestrator (using controller-based attach-detach)
```

Comment:
Am I right that the `nodeSelector` needs to be in `values.yaml` before installing PSO?

If that's the case, is it worth calling it out early; or maybe some instruction users can follow for existing installation?

Reply:
Yeah, it should be in there before installing. Updating them after installation gets a little stickier. I'll add a note.